### ISSUE TITLE
Tensorflow object detection API vs Keras

### ISSUE THREAD

**Issue by balavenkatesh3322** [shubham0204/Multiclass_Object_Classification#3]
Hey,
I am new to machine learning. I need clarification on what is the difference between using Keras instead of tensorflow object detection API. We need to manually configure hidden layers and input layer so what is the advantage and how to know how many layers should configure to achieve object detection using Keras. 

Thanks !!!
